### PR TITLE
Upgrade alembic migration for multiple databases

### PR DESCRIPTION
**Pull request by ncloudioj**
This refers to Bug 1181659 - Create Alembic migrations for new RDS campaign database (postgres)

### CODE REVIEW

File: migrations/env.py [hunk 1]
```diff
@@ -2,6 +2,10 @@
 from alembic import context
 from sqlalchemy import engine_from_config, pool
 from logging.config import fileConfig
+import logging
+import re
+
+USE_TWOPHASE = False
```
Comment (oyiptong) [inline migrations/env.py]: When is USE_TWOPHASE ever true?

Comment (ncloudioj) [inline migrations/env.py]: Good catch @oyiptong , I've also noticed that. We don't need the two phase commit here though. This option is from the Alembic template for the multi-db scenario, I didn't remove it mainly for the compatibility concern. I think it's OK here to keep all its default options. What do you think?

Comment (oyiptong) [inline migrations/env.py]: I think it's ok

**Comment by oyiptong** [comment]
r+

**Pull request closed without merging**